client/packages/requisitions/src/ResponseRequisition/DetailView/Toolbar.tsx

```diff
@@ -19,12 +20,14 @@ export const Toolbar: FC = () => {
   const t = useTranslation(['distribution', 'common']);
   const isDisabled = useResponse.utils.isDisabled();
   const { itemFilter, setItemFilter } = useResponse.line.list();
-  const { otherParty, theirReference, update } = useResponse.document.fields([
+  const { otherParty, theirReference, shipments, update } = useResponse.document.fields([
     'lines',
     'otherParty',
     'theirReference',
+    'shipments'
   ]);
   const { onDelete } = useResponse.line.delete();
+  const linkedToShipment = shipments?.totalCount > 0;
```

Comment:
this is personal preference.. because you are checking for `!linkedShipment` later - it would be neater to have the state reversed here:

```suggestion
  const noLinkedShipments = (shipments?.totalCount ?? 0) === 0;
```